room.h

```diff
@@ -90,6 +90,7 @@ namespace QMatrixClient
             QList<User*> membersLeft() const;
 
             Q_INVOKABLE QList<User*> users() const;
+			Q_INVOKABLE QStringList usernames() const;
```

Comment:
That didn't help, I added `Q_DECLARE_METATYPE(User*)` there, there is also `qRegisterMetaType<User*>    ("User*");` but the error ends up being the same.

I could implement it as mentioned above, with a `memberNames()` function (instead of `usernames()`)